Symfony/CS/Fixer/Symfony/SpacesAfterSemicolonFixer.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Token;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class SpacesAfterSemicolonFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        for ($index = count($tokens) - 3; $index > 0; --$index) {
```

Comment:
yes, 1 because it is the count, 1 because we look ahead one token in each iteration, and one because if the semicolon is the last of the tokens we don't want to add space after it anyway (`eof_ending` will take care of that)